File: 0455-assign-cookies/0455-assign-cookies.js
/**
 * @param {number[]} g
 * @param {number[]} s
 * @return {number}
 */
var findContentChildren = function(g, s) {
    let c = 0;
    g.sort((a, b) => a - b);
    s.sort((a, b) => a - b);

    for (let v of s) {
        if (v >= g[c]) {
            c++;
            if (c === g.length) break;
        }
    }

    return c
};

File: 0455-assign-cookies/0455-assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        count = 0
        g.sort()
        s.sort()
        
        for c in s:
            if c >= g[count]:
                count += 1
                if count == len(g):
                    break
        
        return count